easybuild/easyconfigs/t/TAMkin/TAMkin-1.0.8-intel-2015b-Python-2.7.10.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "PythonPackage"
+
+name = 'TAMkin'
+version = '1.0.8'
+
+homepage = 'http://molmod.github.io/tamkin/'
+description = """TAMkin is a post-processing toolkit for normal mode analysis,
+ thermochemistry and reaction kinetics. It uses a Hessian computation from a
+ standard computational chemistry program as its input."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
```

Comment:
Indeed, we need a match with several other packages. I'm in favor of upgrading to 2016a another time though. I should then do this consistently for a whole bunch of packages.